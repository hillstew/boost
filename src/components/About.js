import React from 'react';

export const About = () => {
  return <p>a lot of stuff of about be the boost</p>;
};
