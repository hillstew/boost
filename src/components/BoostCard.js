import React from 'react';
import { Link } from 'react-router-dom';
import PropTypes from 'prop-types';

export const BoostCard = ({ img, fetchDogImage }) => {
  const url = 'https://random.dog/';
  return (
    <div className="BoostCard">
      <Link to={`/send-boost/${img}`}>
        <img src={url + img} width="300px" alt="dog" />
      </Link>
      <button className="BoostCard-button" onClick={() => fetchDogImage()} />
    </div>
  );
};

BoostCard.propTypes = {
  img: PropTypes.string,
  fetchDogImage: PropTypes.func
};
