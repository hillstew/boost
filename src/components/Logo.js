import React from 'react';
import logo from '../images/betheboost.png';

export const Logo = () => {
  return (
    <footer className="Logo-footer">
      <img src={logo} alt="be the boost logo" height="200px" />
    </footer>
  );
};
