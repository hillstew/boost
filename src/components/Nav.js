import React from 'react';
import { NavLink } from 'react-router-dom';

export const Nav = () => {
  return (
    <nav className="Nav">
      <NavLink to="/saved">Saved</NavLink>
      <hr />
      {/* <NavLink to="/sent">Sent</NavLink>
      <hr /> */}
      <NavLink to="/about">About</NavLink>
      <hr />
    </nav>
  );
};
