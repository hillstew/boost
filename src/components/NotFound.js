import React from 'react';

export const NotFound = () => {
  return <p>not found</p>;
};
