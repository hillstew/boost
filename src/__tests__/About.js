import React from 'react';
import { About } from '../components/About';
import { shallow } from 'enzyme';

describe('About', () => {
  it('should match the snapshot', () => {
    let wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
