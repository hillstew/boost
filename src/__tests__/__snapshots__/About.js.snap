// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match the snapshot 1`] = `
<div
  className="About"
>
  <h1>
    About Be the Boost 
  </h1>
  <p>
    Sometimes in life, you find it hard to encourage your friend going through a hard time, of having a bad day. Boost is a way to send your friend a picture of a dog (cats coming soon!) and message.
  </p>
  <img
    alt="good news is coming"
    src="good-things.jpg"
  />
</div>
`;
